update.sh

```diff
@@ -2,13 +2,13 @@
 set -eo pipefail
 
 # Dockerfiles to be generated
-versions="2.4.0-snapshot 2.3.0 2.2.0 2.1.0 2.0.0 1.8.3"
+versions="2.4.0.M3 2.4.0-snapshot 2.3.0 2.2.0 2.1.0 2.0.0 1.8.3"
 arches="i386 amd64 armhf arm64"
 bases="debian alpine"
 
 # Generate header
 print_header() {
-	cat > $1 <<-EOI
+    cat > $1 <<-EOI
```

Comment:
Code wise I think it's now all OK!
I'd just replace these 4 spaces with a tab so it is similar to the other methods. ;-)

Reply:
OK, fixed